plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -814,8 +820,10 @@ add_addr_info(struct sockaddr_storage *service, socklen_t *addrlen, AVal *host,
         *socket_error = WSANO_DATA;
 #elif __FreeBSD__
         *socket_error = ENOATTR;
-#else
```

Comment:
Why is this not special cased to openBSD?

Reply:
Because the `#defined(ENODATA)` check is more reliable. The "if platform has the [unportable] `ENODATA` constant, it will be used, otherwise we fall back to `EAFNOSUPPORT`" should work anywhere.